### ISSUE TITLE
Migration directory error

### ISSUE THREAD

**Issue by nurasto** [phalcon/phalcon-devtools#310]
Got this following warning when accessing Migration Menu

>Warning: mkdir(): No such file or directory in E:\xampp\htdocs\phalcon\vendor\phalcon\devtools\scripts\Phalcon\Web\Tools\controllers\MigrationsController.php on line 34

Seems WebTools working directory at runtime is not under project directory. I wish MigrationController.php could read migration directory configuration defined in config.php/config.ini.

**Comment by alexglue** [comment]
There is a hardcoded value in MigrationController.php:
```
    protected function _getMigrationsDir()
    {
        $migrationsDir = 'app/migrations'; //todo: get this from config. set correct path!
```
which does not exists on module-style project initialization
```
phalcon project PROJECT_NAME modules --enable-webtools
```
but there is an apps folder, and looks like you should to get current path from _getMigrationsDir()
or use from-config value.

**Comment by austerus** [comment]
strangely, even when I manually create "migrations" folder (writable by the webserver user) under "app" (and next to "config" and the others), the error remains so I am unsure where the webtools app is actually looking for that folder

**Comment by alexglue** [comment]
If your project was initialized with --module project-type option, then you should use _apps_ path! not app.

**Comment by sergeyklay** [comment]
Fixed in 2.0.x
